scalafix-tests/unit/src/test/scala/scalafix/tests/SymbolMatcherTest.scala

```diff
@@ -23,6 +23,7 @@ class SymbolMatcherTest extends BaseSemanticTest("SymbolMatcherTest") {
   test("normalized") {
     val term = SymbolMatcher.normalized(Symbol("_root_.Foo.a."))
     assert(term.matches(Symbol("_root_.Foo.a#"))) // type
+    assert(term.matches(Symbol("_root_.Foo.a.apply()LFoo/a;."))) // apply
```

Comment:
Nice, this should have been documented in the tests 👍 Although, I am OK with changing `A.apply(...)` to normalize to `A.apply` only, like other methods.